prow/tide/tide_test.go

```diff
@@ -4288,6 +4289,16 @@ func TestQueryShardsByOrgWhenAppsAuthIsEnabledOnly(t *testing.T) {
 			},
 			expectedNumberOfApiCalls: 2,
 		},
+		{
+			name:               "Apps auth is used, one call per org, goroutines limited",
```

Comment:
In the PR comments, Petr noted that you were not using Apps auth - likely you want to add such a test to the matrix?

Reply:
For the concurrency bound it's indifferent. Actually, we won't benefit from changes in ``blockers.go`` and ``status.go`` as they will most likely be collapsed to 1 query. We will benefit from limiting concurrency in ``tide.go`` as every outer loop will have it's own iteration. As I wrote before, for the paged queries it may still be not enough though. Back to this change, if it's indifferent and I am just testing the mechanism, I just want to have more than 1 calls to API concurrently and it was a quicker way to invoke this in test IMHO.